### PR TITLE
fix: fix privilege group list and list collections

### PR DESCRIPTION
**Pull request by shaoting-huang**
related: #37031
* built-in privilege group privileges in listPrivilegeGroups() should be the same as in milvus.yaml
* collections granted by collection level built-in privilege group should be list in showCollections()

### CODE REVIEW

**Comment by shaoting-huang** [comment]
after fix:
```
>>> role.list_grants()
GrantInfo groups:
- GrantItem: <object:Global>, <object_name:*>, <db_name:default>, <role_name:role>, <grantor_name:root>, <privilege:CollectionAdmin>
>>>
>>> role.grant_v2("CollectionAdmin", "*", db_name="default")
>>> role.grant_v2("CreatePrivilegeGroup", "*", db_name="*")
>>>
>>> role.list_grants()
GrantInfo groups:
- GrantItem: <object:Global>, <object_name:*>, <db_name:*>, <role_name:role>, <grantor_name:root>, <privilege:CreatePrivilegeGroup>
- GrantItem: <object:Global>, <object_name:*>, <db_name:default>, <role_name:role>, <grantor_name:root>, <privilege:CollectionAdmin>
>>> client = MilvusClient(uri="http://localhost:19530", user="user1", password="Milvus")
>>> client.list_collections()
['col1']
```

File: pkg/util/paramtable/rbac_param.go [hunk 1]
```diff
@@ -105,10 +205,67 @@ func (p *rbacConfig) init(base *BaseTable) {
...
+		}
+	}
+	return nil
+}
+
+func (p *rbacConfig) GetDefaultPrivilegeGroupNames() []string {
+	return lo.Keys(builtinPrivilegeGroups)
+}
+
+func (p *rbacConfig) IsCollectionPrivilegeGroup(privName string) bool {
```
Comment (tedxu) [inline pkg/util/paramtable/rbac_param.go:258]: nit: it is unusual to write implementation details in configuration declarations.

**Review by tedxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**